src/participants/create_project.py

```diff
@@ -134,10 +138,10 @@ def handle_wi(self, wid):
             return
 
         # events for official projects that are gated get diverted to a side project
-        prjobj = get_or_none(Project, name=project, obs__apiurl=self.obs.apiurl)
+        prjobj = Project.get_matching(project, self.obs.apiurl)
         if prjobj and prjobj.gated:
-            print(prjobj, "is gated")
-            link = project
+            print "%s is gated" % prjobj
```

Comment:
regressing ? I'm just replacing the existing print arguments because the string originally printed is : 
home:lbt "is gated"
(iirc) as we're printing 2 objects and the string object has " around it :)
Pure sugar